client/src/app/services/view-state.service.ts

```diff
@@ -1296,7 +1301,7 @@ export class ViewStateService
                 ), "underspectrum1");
 
                 // Set top widgets to defaults
-                this._viewState.analysisLayout.topWidgetSelectors = [ViewStateService.widgetSelectorContextImage, ViewStateService.widgetSelectorSpectrum];
+                this._viewState.analysisLayout.topWidgetSelectors = [ViewStateService.widgetSelectorContextImage, ViewStateService.widgetSelectorSpectrum, ViewStateService.widgetSelectorTernaryPlot];
```

Comment:
Oh, we're putting a ternary up top but hiding it? Is this a hack to remember the state?

Reply:
Yeah the 3rd top widget selector is the preview container - we could just default this to something if we don't want to store it, but the widget selectors are just stored as a list of strings, so this seemed like the easiest way to get it in.